styles.css

```diff
@@ -1,20 +1,149 @@
-.home{
-position: relative;
-width: 375px;
-height: 6968px;
-background: #FFFFFF;
-}
-
-.cover{
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 120px 24px;
-gap: 12px;
-position: absolute;
-width: 375px;
-height: 590px;
-left: 0px;
-top: 92px;
-background: #1C1A19;
-}
\ No newline at end of file
+
+.home {
+  /* width: 375px; */
+
+  /* height: 6968px; */
+  background: #fff;
+  z-index: 10px;
+}
+
+.cover {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 1px;
+  position: absolute;
+  left: 1px;
+  right: 1px;
+  bottom: 1px;
+  top: 60px;
+  background-color: #1c1a19;
+}
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you also the **background image** as from your chosen Figma template to stick to the expected design. I recommend you add this change as it was missed by the previous reviewer.
  - Also make sure the section the design from the Figma template by using the same `font-family`, `font-size`, `font-weight`, `spacing between and around elements`, `colors` (background and font),... as the ones defined for the template.

| Figma Design | Your Design |
| ------------------- | ----------------- |
| ![home_f2](https://user-images.githubusercontent.com/83042742/185543170-d102e9cb-a3bc-45a5-a423-c7079cd9a98b.png) | ![Screenshot from 2022-08-19 06-27-22](https://user-images.githubusercontent.com/83042742/185543218-73a5a3b6-1192-4b6e-9298-e1b85fffa307.png) |

Reply:
i dont understand how apply this in my css document

Reply:
ok i add some font to my web page hope you can see it